R/utils.R

```diff
@@ -57,3 +57,24 @@ release_bullets <- function() {
     NULL
   )
 }
+
+#' Return the path to one of the packages in the devtools test dir
+#'
+#' Devtools comes with some simple packages for testing. This function
+#' returns the path to them.
+#'
+#' @param package Name of the test package.
+#' @keywords internal
+#' @examples
+#' if (has_tests()) {
+#' devtest("testUseData")
+#' }
+#' @export
+devtest <- function(package) {
```

Comment:
Wonder if we should deprecate this...

Reply:
Or just remove it? Doesn't look like anyone uses it (https://github.com/search?p=2&q=org%3Acran+devtest&type=Code)

Reply:
yeah, I would be fine with that as well, I'll remove it.